### ISSUE TITLE
Scope of Current Extension Effort

### ISSUE THREAD

**Issue by aleclaws** [KantaraInitiative/wg-uma#364]
We've been have a lot of discussion around the proposed/possible extensions. This issue attempts to list those and shortly define what they are trying to accomplish

![UMAthinkin-extensions](https://user-images.githubusercontent.com/2133341/94182294-c7ca4100-fe6e-11ea-9152-45e2ef501ca1.png)

1. Policy Manager "The AS exposes an API to allow the RO to manage policy over registered resources" (there is an editors draft already)
2. Relationship Manager (name?) "The RS exposes and API to allow the RO to manage resource registration/policy at an AS" (parts of this exist on the mailing list)
3. Trusted Claims(Cascading AS in the diagram) "The AS delegates authentication (and maybe claims gathering) to a second AS (UMA or OIDC?)" 
4. Fallback AS "During Resource Registration, The RO can direct the RS to one (or more?) AS"

I've also includes some other possible extensions
5. Resource Definitions "The AS defines general resource registration so that specific RO resources may not need explicit registration, enables AS-first request flows"
6. VC/DID claims profiles "UMA loosly defines IDToken based claims pushing, is there value in a VC claims profile"

For the above the questions/consideration:
- keep independent or make sense to merge some of this? (how many extensions are there?)
- what are the use-cases that support these extensions efforts?
- whats the overlap/intersection with other WGs (HEART,FAPI,GNAP,UDAP?)

**Comment by xmlgrrl** [comment]
I've made my best attempt to try and interpret our opportunities to address new use cases (for a new presentation), and thought I'd provide the attempt here.
![UMA-new-work](https://user-images.githubusercontent.com/248613/94814101-81b93400-03be-11eb-8183-ad0bf610bd4e.jpg)

**Comment by aleclaws** [comment]
Option 3 is the issue: https://github.com/KantaraInitiative/wg-uma/issues/260
Option 4 is an implementation consideration, not for specification

**Comment by xmlgrrl** [comment]
Yes, as reconfirmed in [UMA telecon 2020-10-01](https://kantarainitiative.org/confluence/display/uma/UMA+telecon+2020-10-01).
